src/components/Loading/Loading.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+
+import Loading from './Loading';
+
+it('renders without crashing', () => {
+<<<<<<< HEAD
```

Comment:
It looks like this was added by a merge or rebase conflict. We need to get rid of it before merging.